google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -265,6 +341,86 @@ public Row readRow(String tableId, String rowKey) {
     return readRowsCallable().first().futureCall(Query.create(tableId).rowKey(rowKey));
```

Comment:
Please call `readRowAsync(tableId, rowKey, null)` here and the other singular readRow methods that don't take a filter